docs/building_blocks.md

```diff
@@ -8,20 +8,20 @@ Consists of a Provider Framework, a Lookup Framework, a Workflow Engine, and pol
 A Workflow describes a collection of Activities and must be declared in such a way that all input requirements of all contained activities have the potential to be fulfilled within the workflow itself.
 The workflow engine loads this declaration, validates it, and invokes its activities in an order determined by their input and output declarations. The workflow can be packaged into a deployable unit (coming soon!).
 
-## Activity
-An Activity is an abstract executable unit, analog to “a function in a program”, that consumes inputs and produces outputs. The workflow engine triggers its execution when all its input requirements are met. When several activities are grouped into a workflow, the execution in turn produces output available as input to other activities which then get triggered. In essence, this means that all ordering of activities and all concerns regarding parallel or sequential execution, can be expressed as input requirements and output declarations that fulfills them.
+## Step
+A Step is an abstract executable unit, analog to “a function in a program”, that consumes inputs and produces outputs. The workflow engine triggers its execution when all its input requirements are met. When several activities are grouped into a workflow, the execution in turn produces output available as input to other activities which then get triggered. In essence, this means that all ordering of activities and all concerns regarding parallel or sequential execution, can be expressed as input requirements and output declarations that fulfills them.
```

Comment:
Should we mention here that inputs are called parameters and outputs are called returns?